steamtinkerlaunch

```diff
@@ -7433,7 +7433,7 @@ function installd3d47dll {
 }
 
 function createDLReShadeList {
-	RESHADEVERSIONS="5.4.2!5.0.0!4.2.0!3.5.3" # List some hardcoded reshade versions to selct for the override 
+	RESHADEVERSIONS="5.0.0!4.9.1!3.4.1" # List some hardcoded reshade versions to selct for the override 
```

Comment:
that actually ended up being my solution to the override problem I set the first value to `$RSVERSOVRD`  that way it will overwrite itself with itself :) 

then I removed the default value from the yad menu command (as that was also overriding and since I am always initializing `RSVERSOVRD` to be the same as `RSVERS` it will by default auto-populate `RSVERS` 

I'll push this change in a bit so you can review it when you have time